packages/@glimmer/runtime/lib/compiled/opcodes/debugger.ts

```diff
@@ -25,48 +25,48 @@ let callback = debugCallback;
 
 // For testing purposes
 export function setDebuggerCallback(cb: DebugCallback) {
-  callback = cb;
+  if (import.meta.env.DEV) callback = cb;
```

Comment:
since this function is now a no-op in prod, would terser propagate the "nothingness" that then would be this function, and remove all the invocations?

Reply:
Yes indeed.